app/views/reasons/CannotDiscloseImportVATView.scala.html

```diff
@@ -0,0 +1,61 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(layout: templates.Layout,
+      formHelper: FormWithCSRF,
+      h1: components.h1,
+      p: components.p,
+      bullets: components.bullets,
+      linkButton: components.linkButton
+      )
+
+@(isRepFlow: Boolean, importerName: Option[String])(implicit request: Request[_], messages: Messages)
+
+@layout(
+        pageTitle = messages("CannotDiscloseImportVAT.pageTitle"),
+        customBackLinkUrl = None,
+        showBackLink = false
+        ) {
+
+    @h1(messages("CannotDiscloseImportVAT.pageTitle"))
+
+    @if(isRepFlow){
+        @p {
+            @messages("CannotDiscloseImportVAT.rep.p1", importerName.get)
+           }
+        }  else {
+        @p {
```

Comment:
the formatting here is a bit odd - I think the @p elements should be indented more (so that we can distinguish `if` braces from @p braces)